clogMonitor/src/components/TimeRange.js

```diff
@@ -15,7 +16,56 @@ import { FormControl, FormHelperText, InputLabel, OutlinedInput, Stack } from '@
  * 
  * @returns {React.ElementType}
  */
-const TimeRange = ({ startTime, startChangeHandler, endTime, endChangeHandler, direction="column" }) => {
+
+function hasDSTeffect(time) {
+    // check to see if DST is in effect in user's timezone 
+    const parsedDate = new Date(time)
+    const jan = new Date(parsedDate.getFullYear(), 0, 1); 
+    const jul = new Date(parsedDate.getFullYear(), 6, 1); 
+    // standard time offset
+    const stdOffset = Math.max(jan.getTimezoneOffset(), jul.getTimezoneOffset());
+    if(parsedDate.getTimezoneOffset() < stdOffset) {
+        return true;
+    }
+    return false;
+}
+
+export function hasDSTconflict(time) {
+
+    if (!hasDSTeffect(time)) {
+        return false;
+    }
+    // check to see if chosen date/time has DST conflict
+    const parsedDate = new Date(time)
+    let curHour = parsedDate.getHours();
+    let curMinute = parsedDate.getMinutes();
+    //check to see if chosen date/time is the start date/time of DST
+    const curMidnight = new Date(parsedDate.getFullYear(), parsedDate.getMonth(), parsedDate.getDate(), 0).getTimezoneOffset();
+    const cur3AM = new Date(parsedDate.getFullYear(), parsedDate.getMonth(), parsedDate.getDate(), 3).getTimezoneOffset();
+    if (curMidnight < cur3AM && curHour === 1 && curMinute >= 0 && curMinute < 60){
+            return true;
+       }
+    return false;
+}
+
+export function hasDSTerror(time) {
+    // check to see if date/time is not existed (e.g. 2021-3-14 2:30AM)
+    // time format: '2021-11-07T01:30:00' 
+
+    if (!hasDSTeffect(time)) {
+        return false;
+    }
+    const parsedDate = new Date(time)
+    let curHour = parsedDate.getHours();
+    if (curHour!==parseInt(time.substring(11,13))) {
+        return true;
+    }
+    return false;
+}
+
+
+const TimeRange = ({ startTime, startChangeHandler, endTime, endChangeHandler, startTimeDST, startDstChangeHandler, endTimeDST, endDstChangeHandler, direction="column"}) => {
+
     // Error checking
     const isRangeError = () => {
```

Comment:
This is now broken because it doesn't take into account DST. Could you fix it?
For example, 11/07/2021 1:59 (BEFORE) is before 11/07/2021 1:00 (AFTER), but this will only see that 1:00 is before 1:59 and make it look like there's an input error when there isn't. 

If you move convertDSTtoUTC from LogEventsFilters.js to here then you can probably use that to fix this